index.html

```diff
@@ -5,45 +5,859 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>TNW</title>
     <link rel="stylesheet" href="./css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/0eaab31a39.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
+
   <body>
-    <header></header>
-    <section class="display"></section>
-    <article></article>
-    <section class="funding"></section>
-    <main>
-      <article>
-        <div class="main-image">
-            <img src="" alt="" />
-            <h3 class="category"></h3>
-            <h2 class="title"></h2>
-            <h4 class="author-time"></h4>
+    <section class="top-nav">
+      <nav class="relative d-flex">
+        <div class="nav-news">
+          <ul class="d-flex">
+            <li>News</li>
+            <li>Events</li>
+            <li>Business</li>
+            <li>AMAs</li>
+            <li>Spaces</li>
+            <li>Terms & Conditions</li>
+          </ul>
         </div>
-        <figcaption>
-            <h3 class="category"></h3>
-            <h2 class="title"></h2>
-            <img src="" alt="">
-        </figcaption>
-        <ul>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-        </ul>
+        <div class="about">
+          <ul class="d-flex">
+            <li>About</li>
+            <li>Advertise</li>
+            <li>Jobs</li>
+            <li>Contact</li>
+          </ul>
         </div>
-      </article>
-    </main>
-    <section class="latest">
-      <article>
-        <img src="" alt="" />
-        <h2></h2>
-        <h3></h3>
-      </article>
+      </nav>
     </section>
-    <footer>
-      <div class="icons"></div>
-      <nav></nav>
-      <div class="copywrite"></div>
-    </footer>
+    <header class="stk">
+      <div class="mid-nav-main d-flex">
+        <div class="mid-nav-image">
+          <img src="./images/head/tnw.png" alt="image" />
+        </div>
+        <nav class="fixed d-flex mid-nav">
+          <div class="nav-left hd-m">
+            <ul class="d-flex v-middle">
+              <li>LATEST</li>
+              <li>HARD FORK</li>
+              <li>PLUGGED</li>
+              <li>README</li>
+              <li>GROWTH QUARTERS</li>
+              <!-- <li>SHIFT</li>
+              <li>NEURAL</li> -->
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

Reply:
- [ ] Please remove this commented piece of code.